### PR TITLE
Pyet xarray

### PR DESCRIPTION
**Pull request by mvremec**
Update includes:
- compatibility with xarray.DataArray, so pyet can be used with netcdf data,
- improved testing (benchmarked to McMahon et al. 2013 and Schroedter 1985),
- improved documentation,
- added examples, ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,64 +2,101 @@
...
+drainage paper 56](https://www.fao.org/3/x0490e/x0490e00.htm), [McMahon et al., 2013 (supplementary)](https://hess.copernicus.org/articles/17/4865/2013/) and [Schrödter, 1985](https://link.springer.com/book/10.1007/978-3-642-70434-5). In addition, two comparative analysis between daily PE estimated with pyet and other organizations is
+made:
+
+*   `pyet.pm_fao56` against daily PE estimated with ASCE Penman-Monteith from [CoAgMET](https://coagmet.colostate.edu/) (
+  Colorado State University),
+
+*   `pyet.makkink` against daily PE estimated with Makkink from The Royal Netherlands Meteorological
+  Institute ([KNMI](https://www.knmi.nl/over-het-knmi/about)).
+
+## Dimensions
```
Comment (raoulcollenteur) [inline README.md:84]: Make it Pyet 1.2, and what about DataFrames?

File: pyet/version.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # This is the only location where the version will be written and changed.
 # Based on https://packaging.python.org/single_source_version/
-__version__ = "1.1.0"
+__version__ = "1.1.1"
```
Comment (raoulcollenteur) [inline pyet/version.py:3]: This should be 1.2.0b, cause you merge it into Dev ('b') and it is definitely more than a minor update.

**Review by raoulcollenteur — CHANGES_REQUESTED** [review]

**Comment by raoulcollenteur** [comment]
We should really not merge into the Master, but first into Dev. And from there, merge Dev into Master.

**Comment by mvremec** [comment]
Merge first to dew.

**Pull request closed without merging**